2-farm/lessons/5-migrate-application-to-the-cloud/README.md

```diff
@@ -375,7 +375,7 @@ This will create a folder inside the `soil-moisture-trigger` folder called `iot-
     For detailed output, run func with --verbose flag.
     [2021-05-05T02:44:07.517Z] Worker process started and initialized.
     [2021-05-05T02:44:09.202Z] Executing 'Functions.iot-hub-trigger' (Reason='(null)', Id=802803a5-eae9-4401-a1f4-176631456ce4)
-    [2021-05-05T02:44:09.205Z] Trigger Details: PartionId: 0, Offset: 1011240-1011632, EnqueueTimeUtc: 2021-05-04T19:04:04.2030000Z-2021-05-04T19:04:04.3900000Z, SequenceNumber: 2546-2547, Count: 2
+    [2021-05-05T02:44:09.205Z] Trigger Details: PartitionId: 0, Offset: 1011240-1011632, EnqueueTimeUtc: 2021-05-04T19:04:04.2030000Z-2021-05-04T19:04:04.3900000Z, SequenceNumber: 2546-2547, Count: 2
```

Comment:
It's possible this is wrong. Did this output come from buggy external code?

Reply:
Yeah - this is the output of the Azure Functions event hub connector. Probably ok to have a spelling fix, and I'm trying to find the relevant code to get that fixed